instrumentation/net/http/splunkhttp/server_timing_test.go

```diff
@@ -26,21 +25,44 @@ import (
 )
 
 func TestServerTimingMiddleware(t *testing.T) {
+	resp := responseForHandler(func(handler http.Handler) http.Handler { // nolint:bodyclose // Body is not used
+		handler = ServerTimingMiddleware(handler)
+		return otelhttp.NewHandler(handler, "server", otelhttp.WithTracerProvider(oteltest.NewTracerProvider()))
+	})
+
+	assert.Equal(t, http.StatusOK, resp.StatusCode, "should return OK status code")
+	assert.Contains(t, resp.Header["Access-Control-Expose-Headers"], "Server-Timing", "should set Access-Control-Expose-Headers header")
+	assert.Regexp(t, "^traceparent;desc=\"00-[0-9a-f]{32}-[0-9a-f]{16}-01\"$", resp.Header.Get("Server-Timing"), "should return properly formated Server-Timing header")
+}
+
+func TestNewHandler_Default(t *testing.T) {
```

Comment:
```suggestion
func TestNewHandlerDefault(t *testing.T) {
```

Reply:
Will change even though my eyes would even prefer `Test_NewHandler_default` 😄 But I do not have a strong opinion 😉

Reply:
Addressed https://github.com/signalfx/splunk-otel-go/pull/30/commits/33dec16e4a16090a3f0b13b567f4f30593df4daf